airflow/models/baseoperator.py

```diff
@@ -463,6 +460,7 @@ def __rshift__(self, other):
 
         If "Other" is a DAG, the DAG is assigned to the Operator.
         """
+        from airflow.models.dag import DAG
```

Comment:
Are we jumping the gun here and some code below too by importing the DAG class again on each __rshift__ and __lshift__.

I might be missing something as I am not sure how Python deals with importing the class that is already imported.

Reply:
It just checks if module is loaded in modules[] dictionary. So yes - slightly slower than not doing anything but i do no think it should be a problem at parsing time. But it's a good catch indeed.

Reply:
https://stackoverflow.com/questions/32151193/is-there-a-performance-cost-putting-python-imports-inside-functions

Reply:
Should be no problem w/regards to performance.

Reply:
https://stackoverflow.com/questions/32151193/is-there-a-performance-cost-putting-python-imports-inside-functions